File: documentation/user/en/use/api/example/update-existing-entity.cs
evita.UpdateCatalog(
	"testCatalog",
	session => {
		session.GetEntity("Product", 1, AttributeContentAll(), DataInLocalesAll())
			.OpenForWrite()
			.SetAttribute(
				"name", new CultureInfo("en"),
				"ASUS Vivobook 16 X1605EA-MB044W Indie Black"
			)
			.SetAttribute(
				"name", new CultureInfo("de"),
				"ASUS Vivobook 16 X1605EA-MB044W Indie Schwarz"
			)
			.UpsertVia(session);
	}
);

File: documentation/user/en/use/api/example/update-existing-entity.java
evita.updateCatalog(
	"evita",
	session -> {
		session.getEntity("Product", 1, attributeContentAll(), dataInLocalesAll())
			.orElseThrow(
				() -> new IllegalArgumentException("Product `1` not found!")
			)
			.openForWrite()
			.setAttribute(
				"name", Locale.ENGLISH,
				"ASUS Vivobook 16 X1605EA-MB044W Indie Black"
			)
			.setAttribute(
				"name", Locale.GERMAN,
				"ASUS Vivobook 16 X1605EA-MB044W Indie Schwarz"
			)
			.upsertVia(session);
	}
);
